client/pkg/client/client.go

```diff
@@ -135,6 +136,11 @@ func (c *Client) Talos() *talos.Client {
 	return talos.NewClient(c.conn)
 }
 
+// Resource provides access to the resources API.
+func (c *Client) Resource() *resource.Client {
```

Comment:
This name might be a bit confusing for the API consumers - I'd need to know that to get resources I need to use `.COSI()`, and `.Resource()` does something else. Maybe we find a better name?